bin-src/init.ts

```diff
@@ -0,0 +1,142 @@
+import boxen from "boxen";
+import { execaCommand } from 'execa';
+import { findUp } from 'find-up';
+import { writeFile } from 'jsonfile';
+import meow from 'meow';
+import prompts from 'prompts';
+import type { PackageJson } from 'read-pkg-up'
+import readPkgUp from 'read-pkg-up';
+import { getCliCommand, parseNi } from '@antfu/ni';
+
+import noPackageJson from '../node-src/ui/messages/errors/noPackageJson';
+
+export const TestFramework = {
+    STORYBOOK: 'storybook',
+    PLAYWRIGHT: 'playwright',
+    CYPRESS: 'cypress'
+};
+type TestFrameworkType = typeof TestFramework[keyof typeof TestFramework];
+
+export const getPackageManagerInstallCommand = async (args: string[]) => {
+    return await getCliCommand(parseNi, args, { programmatic: true });
+};
+
+const getSBConfigFilePath = async () => {
+    // Walks up directory tree to find nearest Storybook config file.
+    return await findUp(['.storybook/main.ts', '.storybook/main.js', '.storybook/main.tsx', '.storybook/main.jsx', '.storybook/main.mjs', '.storybook/main.cjs'])
+}
+
+export const addChromaticScriptToPackageJson = async ({ packageJson, packagePath }) => {
```

Comment:
Do we have other tooling that edits `package.json` directly? This just feels like something that could be prone to errors.

Reply:
Yes, we do [here](https://github.com/chromaui/chromatic-cli/blob/main/node-src/lib/checkPackageJson.ts). It's pretty simple and is just writing to file. If they already have a chromatic script, it's overwritten. If they do not want the change, it is easy to revert it to what it was before.

Reply:
I ended up leaving as is for now.